src/transceivercontroller/DefaultTransceiverController.ts

```diff
@@ -273,14 +283,29 @@ export default class DefaultTransceiverController implements TransceiverControll
         direction: 'recvonly',
         streams: [this.defaultMediaStream],
       });
+      this.streamIdToTransceiver.set(index, transceiver);
       this.videoSubscriptions.push(index);
       this.logger.info(
         `adding transceiver mid: ${transceiver.mid} subscription: ${index} direction: recvonly`
       );
     }
   }
 
-  private transceiverIsVideo(transceiver: RTCRtpTransceiver): boolean {
+  getMidForStreamId(streamId: number): string {
+    return this.streamIdToTransceiver.get(streamId)?.mid;
+  }
```

Comment:
Make the signature `string | undefined`, 'cos that's what will actually happen. When we turn on strict mode it will complain if it's just `string`.